qiskit/optimization/converters/integer_to_binary.py

```diff
@@ -171,11 +182,16 @@ def _encode_quadratic_coefficients_dict(self, coefficients: Dict[Tuple[str, str]
     def _substitute_int_var(self):
 
         # set objective
-        linear, linear_constant = self._encode_linear_coefficients_dict(
-            self._src.objective.linear.to_dict(use_name=True))
-        quadratic, quadratic_linear, quadratic_constant = \
-            self._encode_quadratic_coefficients_dict(
-                self._src.objective.quadratic.to_dict(use_name=True))
+        linear, linear_constant = self._convert_linear_coefficients_dict(
+            self._src.objective.linear.to_dict(use_name=True)
+        )
+        (
+            quadratic,
+            quadratic_linear,
+            quadratic_constant,
+        ) = self._convert_quadratic_coefficients_dict(
```

Comment:
You don't need to include '()'. The previous line does not include '()'.
Just, `quadratic, quadratic_linear, quadratic_constant = self._convert_quadratic_coefficients_dict(...)`